recipes/sassc/all/conanfile.py

```diff
@@ -0,0 +1,61 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class SasscConan(ConanFile):
+    name = "sassc"
+    license = "MIT"
+    homepage = "libsass.org"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "libsass command line driver"
+    topics = ("Sass", "sassc", "compiler")
+    settings = "os", "compiler", "build_type", "arch"
+
+    build_requires = "autoconf/2.69", "libtool/2.4.6"
+
+    requires = "libsass/3.6.4"
+
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def configure(self):
```

Comment:
```suggestion
    def validate(self):
```

Reply:
see https://github.com/conan-io/conan-center-index/issues/4261 I think we should wait before using `validate`